### PR TITLE
Add a workflow to merge release into master

### PR DESCRIPTION
**Pull request by hsubox76**
Manually triggerable workflow that merges release into master. Won't be able to test it until after next release.

### CODE REVIEW

File: .github/workflows/merge-release-branch.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+              github.repos.merge({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                base: 'master',
+                head: 'release'
+              })
```
Comment (lahirumaramba) [inline .github/workflows/merge-release-branch.yml:22]: nit: new line

File: .github/workflows/merge-release-branch.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  merge_to_master:
+    runs-on: ubuntu-latest
+    # Allow GITHUB_TOKEN to have write permissions
+    permissions:
+      contents: write
+    steps:
+      - name: Merge to master
+        uses: actions/github-script@0.9.0
```
Comment (lahirumaramba) [inline .github/workflows/merge-release-branch.yml:13]: Looks like the latest version is v6 now (includes breaking changes). Worth checking if we can use one of the recent versions here.

File: .github/workflows/merge-release-branch.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: Merge Release Into Master
+
+on: repository_dispatch
+
+jobs:
+  merge_to_master:
+    runs-on: ubuntu-latest
+    # Allow GITHUB_TOKEN to have write permissions
+    permissions:
+      contents: write
```
Comment (lahirumaramba) [inline .github/workflows/merge-release-branch.yml:10]: Is the goal here to grant `GITHUB_TOKEN` with default permissions (without write access) and add write access in the workflow?

File: .github/workflows/merge-release-branch.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+name: Merge Release Into Master
+
+on: repository_dispatch
```
Comment (lahirumaramba) [inline .github/workflows/merge-release-branch.yml:3]: nit feel free to ignore:
We can set` types:` here if we want to restrict the manual trigger even further with a specific `event_type` value in the webhook payload.

```
on:
  repository_dispatch:
    types: [sdk-release-merge]
```

**Review by lahirumaramba — APPROVED** [review]
LGTM! Left a few comments

File: .github/workflows/merge-release-branch.yml (hunk 4 above)
Comment (hsubox76) [reply .github/workflows/merge-release-branch.yml:3]: Will add this when/if we trigger from a webhook. We'll use the button in the Github UI for now.

**Pull request merged**